modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/InfoButtonPropsTransformer.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function InfoButtonPropsTransformer({
```

Comment:
Was this button working before this change? Why would it stop working now?

Reply:
> Was this button working before this change?
Yes is was.
> Why would it stop working now?
I'm guessing it's because of the changes I added to the [ButtonTag](https://github.com/liferay-frontend/liferay-portal/blob/24832635aa7f3b5f2e320bb5b4205460d68bc1c8/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ButtonTag.java)

Reply:
Let's figure out which changes, then... and after that, you might want to document them in [React Rendering in DXP](https://github.com/liferay/liferay-frontend-guidelines/blob/master/dxp/react_rendering.md)

Reply:
Yes, that's what I'm trying to figure out and if I'm able to understand what goes on I'll document that